dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/java2sec/GrantEntry.java

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.classloading.java2sec;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Iterator;
+
+/**
+ *
+ */
+public class GrantEntry {
+    
+    String codeBase;
+    String signedBy;
+    private List permissionEntries = new ArrayList();
+
+    
+    final static String NEW_LINE = System.getProperty("line.separator");
```

Comment:
Looks like we have the following variables define in several classes, can we have a constants class or re-use the same one?

    final static String NEW_LINE = System.getProperty("line.separator");
    final static String QUOTED_STRING = "quoted string";
    final static String PERMISSION_TYPE = "permission type";
    final static String GRANT_KEYWORD = "grant";
    final static String KEYSTORE_KEYWORD = "keystore";
    final static String CODEBASE_KEYWORD = "codeBase";
    final static String PERMISSION_KEYWORD = "permission";
    final static String SIGNEDBY_KEYWORD = "signedBy";
    final static String FILTER_KEYWORD = "filterMask";

Reply:
@utle, a Constants.java is created now to define all the constants above.    Classes referring to those constants are updated to use Constants.XYZ vs XYZ where XYZ is defined with the class.